CHANGELOG.md

```diff
@@ -0,0 +1,132 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+This file follows the formats and conventions from [keepachangelog.com]
+
+Find Sardana issues in the following locations:
+[old SF sardana issues](https://sourceforge.net/p/sardana/tickets/)
+[migrated sardana issues](https://github.com/sardana-org/sardana/issues)
+
+
+## [Unreleased]
+
+### Added
+- bumpversion support
+- This CHANGELOG.md file
+
+
+## [2.1.1] - 2016-09-27
+For a full log of commits between versions run (in your git repo):
+`git log 2.1.0..2.1.1`
+
+### Fixed
+Fix SF issues: #509, #507, #508, #511, #426
+
+
+## [2.1.0] - 2016-09-13
+For a full log of commits between versions run (in your git repo):
+`git log 2.1.0..2.0.0`
+Main improvements since sardana 2.0.0 (aka Jan16) are:
+
+### Added
+- Add itango dependency to make Sardana compatible with the latest
```

Comment:
IMHO, these entries need to be rewritten to avoid "added" or "add" in the "Added setcion. It is unnatural and redundant. Similarly, avouid "fix" or "solve"  in the case of the "Fixed" sections, etc.